docs/marketplace/user/merchant-portal-user-guides/202204.0/products/concrete-products/managing-marketplace-concrete-product.md

```diff
@@ -34,42 +33,43 @@ By default, the table is sorted descendingly by SKU.
 ## Activating and deactivating a concrete product
 
 To activate a marketplace concrete product:
-1. On the **Concrete Products** page, next to the concrete product you want to activate, hover over the three dots and click **Manage Product** or just click the line. The **Concrete Product SKU, Name** page opens.
-2. In the **Status** pane, select **Concrete Product is online** to make the concrete product online.
+
+1. On the *Concrete Products* page, next to the concrete product you want to activate, hover over the three dots and click **Manage Product** or just click the line. This takes you to the *Concrete Product SKU, Name* page.
+2. In the *Status* pane, select *Concrete Product is online* to make the concrete product online.
 3. Click **Save**.
 
-To deactivate the product, clear **Concrete Product is online** to make the concrete product offline.
+To deactivate the product, deselect *Concrete Product is online* to make the concrete product offline.
```

Comment:
Please revert to "clear": https://developers.google.com/style/word-list#letter-d